creator-node/src/models/audiususer.js

```diff
@@ -1,16 +1,10 @@
 'use strict'
 module.exports = (sequelize, DataTypes) => {
   const AudiusUser = sequelize.define('AudiusUser', {
-    audiusUserUUID: {
-      type: DataTypes.UUID,
-      allowNull: false,
-      primaryKey: true,
-      defaultValue: DataTypes.UUIDV4
-    },
     blockchainId: {
       type: DataTypes.BIGINT,
       unique: true,
-      allowNull: true
+      allowNull: true // should this be false?
```

Comment:
i think since we dropped the two-part system of POST /audius_users -> POST /audius_users/associate, we can enforce non-null on this

Reply:
okay i can change this as well